EngineLayer/FdrAnalysis/PEPValueAnalysisGeneric.cs

```diff
@@ -13,13 +14,29 @@ namespace EngineLayer
 {
     public static class PEP_Analysis
     {
+        private static readonly string filepathSubstitue = "SubstitueFilePath";//this is only for unit test that have psms with no filename.
+
         public static string ComputePEPValuesForAllPSMsGeneric(List<PeptideSpectralMatch> psms)
         {
-            Dictionary<string, int> accessionAppearances = GetAccessionCounts(psms);
+            string searchType = DetermineSearchType(psms);
+
+            //These two dictionaries contain the average and standard deviations of hydrophobicitys measured in 1 minute increments accross each raw
+            //file separately. An individully measured hydrobophicty calculated for a specific PSM sequence is compared to these values by computing
+            //the z-score. That z-score is used in the the machine learning.
+            //Separate dictionaries are created for peptides with modifications becuase SSRcalc doesn't really do a good job predicting hyrophobicity
+            Dictionary<string, Dictionary<int, Tuple<double, double>>> fileSpecificTimeDependantHydrophobicityAverageAndDeviation_unmodified = new Dictionary<string, Dictionary<int, Tuple<double, double>>>();
```

Comment:
a comment here describing what each element is would be nice. I think this is:
string - filename
int - rounded RT
double - ?? hydrophobicity?
double - std dev RT?

Reply:
this is sort of nitpicky but it should probably be "The value Tuple is the average and standard deviation, respectively, of the predicted hydrophobicities of the observed peptides eluting at that rounded retention time"